opm/input/eclipse/Schedule/Group/Group.cpp

```diff
@@ -865,6 +865,14 @@ bool Group::has_gpmaint_control(ProductionCMode control) const {
 
 }
 
+bool Group::as_choke() const {
+    return this->m_choke_group.has_value();
```

Comment:
Is it better with has? I also see the point of making it consistent with  NODEPROP::AS_CHOKE so no strong opinions.